src/ROUTES.ts

```diff
@@ -573,6 +573,10 @@ const ROUTES = {
         route: 'workspace/:policyID/tags',
         getRoute: (policyID: string) => `workspace/${policyID}/tags` as const,
     },
+    WORKSPACE_TAG_CREATE: {
+        route: 'workspace/:policyID/tag/new',
```

Comment:
This route should use plural to be consistent with the other pages
```suggestion
        route: 'workspace/:policyID/tags/new',
```

Reply:
Handled.